src/common/Tabs/tabs.tsx

```diff
@@ -25,6 +25,7 @@ const TabsComponent: React.FC<TabsComponentPropsModal> = ({
   const OSC = localStorage.getItem("name") === "OSC";
   const DSEP = localStorage.getItem("name") === "DSEP";
   const UEI = localStorage.getItem("name") === "UEI";
+  const dsepUnified = localStorage.getItem("name") === "dsepUnified";
```

Comment:
change this variable name to isDsepUnified

Reply:
Done